### DISCUSSION TITLE
Cant Verify because My Document Has 2 Sides.

### DISCUSSION THREAD

**Discussion by rxbyydev** [community/community#139633] [GitHub Education]
### Hi everyone,

Hi,

My Document has 2 Sides And On the front there is the date and in the inside there are my data

HOW THE F.
Am i supposed to Put 2 pictures to github if only one works?

**Comment by SebastienXia** [comment]
Hello, @rxbyy 

You can take two pictures of both side and do some PhotoShop or something like this software, then you can upload the document with the both sides. 🤔​

**Reply by rxbyydev** [reply]
Well but that will probably make the file size Too Much?

**Reply by shubham-iot** [reply]
Thanks for your response @SebastienXia . I have both the side in one image but I can't see option to upload, I can only see take picture. Could you please tell, how to upload image?

Thanks,
SJ

**Comment by SebastienXia** [comment]
You can use this web site : to compress your file size. https://www.iloveimg.com/fr to compress your file size.

**Comment by JuanCon1** [comment]
First, scan each side of your ID and save them as a jpg file (photo). Then add them to a Word file and print it at the highest resolution so that when you take a screenshot you will see both sides.
